internal/cli/cli.go

```diff
@@ -41,10 +48,100 @@ const (
 	CheckResultTagError   = Red + "ERROR" + Reset
 )
 
+var CliCheckLogSniffer = &cliCheckLogSniffer{}
+
+var (
+	isTTY bool
+
+	checkLoggerChan chan string
+	stopChan        chan bool
+)
+
+func init() {
```

Comment:
Does this call need to be done as part of an init()?

Reply:
The alternatives don't look quite good, either, but I'll try to change it in a further PR, since I want to improve the non-tty output.